v3/setup/splunk.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
so this file might not even be needed since we can just add it into the `cluster` configs. [Here is the example](https://github.com/adobe-platform/infrastructure/blob/master/v2/config/example-stack-all/secrets-example.json#L56). Like I was saying, if the configs are added correctly, the credential download script will automatically add them into etcd.